javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,27 @@
+function eventDay(noOfdaysFromToday) {
```

Comment:
This function does the job. But I have to say it is not easy to read the code. I don't see the purpose of having two objects for month and weekday. while you could use array. Also this function prints out a lot of things, while in the question's example you need to only print out few words . I would encourage you rewrite the function and think about that someone has to read it in two years and understand it, it will also be a part project which might have thousands line of codes, so less lines of codes means faster application.